### PR TITLE
Rm covalent

### PR DESCRIPTION
**Pull request by suprishi**
Removed covalent and upgraded to version 8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,9 +62,10 @@
     "karma-firefox-launcher": "^1.1.0",
     "karma-jasmine": "^2.0.1",
     "karma-jasmine-html-reporter": "^1.4.0",
-    "protractor": "^5.4.2",
-    "ts-node": "^7.0.1",
+    "protractor": "~5.4.2",
+    "redux-devtools-extension": "^2.13.7",
```
Comment (sjimenez77) [inline package.json]: The `redux-devtools-extension` dependency is not necessary and we should remove it in the `package.json` file.

**Review by sjimenez77 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (suprishi) [reply package.json]: Removed it. Sorry for responding late, I noticed this just now.

**Review by sjimenez77 — APPROVED** [review]

**Comment by sjimenez77** [comment]
Hi @suprishi, I am checking the application and it is not loading the data:

image

The Chrome DevTools show the following warning:

image

And in the backend seems that any api endpoint is triggered when loading data. 
I am using the latest backend sample already fixed that @jdiazgon sent me to test the other Angular template without NgRx. 

Could any of you check it please?

Thanks!

**Comment by suprishi** [comment]
> Hi @suprishi, I am checking the application and it is not loading the data:
> 
> image
> 
> The Chrome DevTools show the following warning:
> 
> image
> 
> And in the backend seems that any api endpoint is triggered when loading data.
> I am using the latest backend sample already fixed that @jdiazgon sent me to test the other Angular template without NgRx.
> 
> Could any of you check it please?
> 
> Thanks!

I was working with JWT authentication on this one. Made corrective changes for CSRF based authentication. It should work now.

**Pull request merged**